go-controller/pkg/config/config.go

```diff
@@ -1183,6 +1183,17 @@ func buildOvnAuth(exec kexec.Interface, northbound bool, cliAuth, confAuth *OvnA
 		return nil, err
 	}
 
+	// When instructed to watch-endpoint, just set scheme and obtain address(es) from endpoint
+	if address == "watch-endpoint" {
```

Comment:
BTW, this will only work if `MasterHA.ManageDBServers` is set to true. Otherwise, we do not start the Endpoint watchers.